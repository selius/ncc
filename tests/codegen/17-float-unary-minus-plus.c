int main()
{
	__print_float(-5.67);
	__print_float(+3.98);
	return 0;
}
