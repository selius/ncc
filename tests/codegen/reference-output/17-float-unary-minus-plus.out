-5.670000
3.980000
